src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(Color color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: " + FigureName.SQUARE.name()
+                + ", area: " + getArea()
+                + ", sq.units, side: " + side
+                + ", units, color: " + getColor());
```

Comment:
[To get a string representation of an enum use `name()`](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)